### PR TITLE
[PDE-4035] chore(cli, core, schema) release 14.0.1

### PR DESCRIPTION
**Pull request by rnegron**
Changelog: 
## 14.0.1

_released `2023-05-22`_

### cli

- :bug: Fix duplicate authentication field keys being generated by the `zapier convert` command ([#639](https://github.com/zapier/zapier-platform/pull/639))
- :bug: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command ([#640](https://github.com/zapier/zapier-platform/pull/640))
- :scroll: Improve docs related to expected behaviors for stale authentication ([#647](https://github.com/zapier/zapier-platform/pull/647))

### core

- None!

### schema

- :scroll: Deprecate the `important` key of the `BasicDisplaySchema`. This property will be removed and hence a breaking change ❗ in the next major version release. ([#644](https://github.com/zapier/zapier-platform/pull/644))

### misc

- :hammer: Dependency updates
  - Bump `moment` from 2.24.0 to 2.29.4 ([#635](https://github.com/zapier/zapier-platform/pull/635))
  - Bump `minimatch` from 3.0.4 to 3.0.5 ([#634](https://github.com/zapier/zapier-platform/pull/634))

### CODE REVIEW

**Review by gregilo — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+## 14.0.1
+
+_released `2023-05-22`_
+
+### cli
+
+- :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command
```
Comment (gregilo) [inline CHANGELOG.md:7]: We typically provide a link to the PR for each item in the CHANGELOG.
```suggestion
- :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command [#640](https://github.com/zapier/zapier-platform/pull/640)
```

Comment (rnegron) [reply CHANGELOG.md:7]: Oops, that's right! I've been playing around with auto-generating these and at this point I've forgotten how they are supposed to look like 😵 Will add this 🙏🏻

**Review by gregilo — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
+## 14.0.1
+
+_released `2023-05-22`_
+
+### cli
+
+- :scroll: Fix duplicate authentication field keys being generated by the `zapier convert` command ([#639](https://github.com/zapier/zapier-platform/pull/639))
+- :scroll: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command ([#640](https://github.com/zapier/zapier-platform/pull/640))
```
Comment (eliangcs) [inline CHANGELOG.md:8]: These two are bug fixes 🐛 

```suggestion
- :bug: Fix duplicate authentication field keys being generated by the `zapier convert` command ([#639](https://github.com/zapier/zapier-platform/pull/639))
- :bug: Fix missing `key` property in the `.zapierapprc` file after running the `zapier convert` command ([#640](https://github.com/zapier/zapier-platform/pull/640))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+- :scroll: Improve docs related to expected behaviors for stale authentication ([#647](https://github.com/zapier/zapier-platform/pull/647))
+
+
+### core
+
+- None!
+
+### schema
+
+- :exclamation: Deprecate the `important` key of the `BasicDisplaySchema`. This property will be removed in the next major version release. ([#644](https://github.com/zapier/zapier-platform/pull/644))
```
Comment (eliangcs) [inline CHANGELOG.md:18]: This is more like a document update, so I would stick with the rule and use 📜. But I understand we probably want to make it more visible, so maybe we can do ❗ in the sentence?

```suggestion
- :scroll: Deprecate the `important` key of the `BasicDisplaySchema`. This property will be removed and hence a breaking change ❗ in the next major version release. ([#644](https://github.com/zapier/zapier-platform/pull/644))
```

**Review by eliangcs — APPROVED** [review]

**Pull request merged**